agent/CMakeLists.txt

```diff
@@ -41,7 +41,7 @@ endif()
 include(multiap-helpers)
 add_definitions(-DBEEROCKS_VERSION="${${PROJECT}_VERSION_STRING}")
 add_definitions(-DBEEROCKS_BUILD_DATE="${BUILD_DATE}")
-add_definitions(-DBEEROCKS_REVISION="${${PROJECT}_REVISION}")
```

Comment:
I think a preparative commit which moves all these definitions to the top level CMakeLists.txt makes more sense. Then we can move it to the multiap_helpers.cmake and be done with it.

Reply:
Meh, that's a lot of rebase work :-)

I wasn't sure about moving it up because it's only used in two components.

Reply:
should have been a (nitpick) anyway, don't worry about it